src/modules/edc-demo/components/empty-state/empty-state.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component, HostBinding, Input} from '@angular/core';
+
+@Component({
+  selector: 'edc-demo-empty-state',
+  templateUrl: './empty-state.component.html',
+})
+export class EmptyStateComponent {
+  @HostBinding('class.flex')
+  @HostBinding('class.flex-col')
+  @HostBinding('class.justify-center')
+  @HostBinding('class.min-h-[35rem]')
```

Comment:
This class for our layouting hack should be applied where the components are used, with class="min-h-[35rem]".

The class uppercase is missing